assets/src/block-validation/amp-validation-status/status-notification.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, ExternalLink } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import AMPValidationErrorsKeptIcon from '../../../images/amp-validation-errors-kept.svg';
+import { BLOCK_VALIDATION_STORE_KEY } from '../store';
+import { StatusIcon } from '../icon';
+import { SidebarNotification } from '../../block-editor/components/sidebar-notification';
+
+/**
+ * AMP validation status notification component.
+ */
+export default function AMPValidationStatusNotification() {
+	const { autosave, savePost } = useDispatch( 'core/editor' );
+
+	const {
+		ampCompatibilityBroken,
+		fetchingErrorsRequestErrorMessage,
+		hasValidationErrors,
+		isDraft,
+		isEditedPostNew,
+		isFetchingErrors,
+		reviewLink,
+	} = useSelect( ( select ) => ( {
+		ampCompatibilityBroken: select( BLOCK_VALIDATION_STORE_KEY ).getAMPCompatibilityBroken(),
+		fetchingErrorsRequestErrorMessage: select( BLOCK_VALIDATION_STORE_KEY ).getFetchingErrorsRequestErrorMessage(),
+		hasValidationErrors: select( BLOCK_VALIDATION_STORE_KEY ).getValidationErrors()?.length > 0,
+		isDraft: [ 'draft', 'auto-draft' ].indexOf( select( 'core/editor' )?.getEditedPostAttribute( 'status' ) ) !== -1,
```

Comment:
Any reason for using the optional chaining operator here? Would `getEditedPostAttribute()` not always be available?

Reply:
You're right – we don't need the optional chaining here.

Reply:
Addressed in cb830a0.